dist/angular-bootstrap-toggle.js

```diff
@@ -94,12 +94,29 @@
 			})
 		.controller('ToggleController',
 			[
-				'$scope', '$attrs', '$interpolate', '$log', '$document', 'toggleConfig', '$toggleSuppressError',
-				function ($scope, $attrs, $interpolate, $log, $document, toggleConfig, $toggleSuppressError) {
+				'$scope', '$attrs', '$interpolate', '$log', '$document', 'toggleConfig', '$toggleSuppressError', '$parse',
+				function ($scope, $attrs, $interpolate, $log, $document, toggleConfig, $toggleSuppressError, $parse) {
 
 					// This controller (self)
 					var self = this;
 
+                    // Support 'ng-true-value' and 'ng-false-value' attrs
+                    var trueValue = parseConstantExpr($parse, $scope, 'ngTrueValue', $attrs.ngTrueValue, true);
+                    var falseValue = parseConstantExpr($parse, $scope, 'ngFalseValue', $attrs.ngFalseValue, false);
+
+                    function parseConstantExpr($parse, context, name, expression, fallback) {
+                        var parseFn;
+                        if (angular.isDefined(expression)) {
+                            parseFn = $parse(expression);
+                            if (!parseFn.constant) {
+                                throw angular.ngModelMinErr('constexpr', 'Expected constant expression for `{0}`, but saw ' +
```

Comment:
jshint is complaining the line is too long, which breaks the build. Could you please fix this? (yeah, no biggie... ;-) )

Reply:
I fixed. :)